src/api/spec/models/configuration_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+RSpec.describe Configuration do
+  it { should validate_presence_of(:name) }
+  it { should validate_presence_of(:title) }
+  it { should validate_presence_of(:description) }
+
+  it 'creates a new Configuration if there is no Configuration' do
+    Configuration.first.destroy
+    expect(Configuration.count).to eq(0)
+    Configuration.title
+    expect(Configuration.count).to eq(1)
+  end
+
+  it 'uses the first Configuration if there is one' do
+    # The first configuration is created by db/seeds.rb
+    expect(Configuration.count).to eq(1)
+    Configuration.title
+    expect(Configuration.count).to eq(1)
```

Comment:
That we still have only one Configuration after calling the method_missing method